qcengine/programs/empirical_dispersion_resources.py

```diff
@@ -637,7 +637,8 @@
             "pbe": {"params": {"s6": 1.000, "s8": 0.000000, "sr6": 2.340218, "beta": 0.129434}},
             "pbe0": {"params": {"s6": 1.000, "s8": 0.000081, "sr6": 2.077949, "beta": 0.116755}},
             "wpbe": {"params": {"s6": 1.000, "s8": 1.280619, "sr6": 1.366361, "beta": 0.003160}},  # formerly lcwpbe
-            "sapt0": {"params": {"s6": 1.000, "s8": 1.475738, "sr6": 1.275350, "beta": 0.039006}},  # experimental
+            "sapt0": {"params": {"s6": 1.000, "s8": 0.88551674, "sr6": 1.38321381, "beta": 0.07548827}},  # JBS 2021
+            "hf": {"params": {"s6": 1.000, "s8": 0.88551674, "sr6": 1.38321381, "beta": 0.07548827}},  # JBS 2021
```

Comment:
are all these digits significant? if you know how many digits you'll report in the paper, worth setting it now.

hf == sapt0?

Reply:
I've used these values so far, but I really doubt all digits are needed. I'll check if going to 6 makes a difference. Also, without dispersion, SAPT0 becomes a HF interaction energy, so the -D parameters are the same for both.